emission/analysis/plotting/composite_trip_creation.py

```diff
@@ -10,26 +10,28 @@
 
 
 def create_composite_trip(ts, ct):
+    isUntrackedTime = ct["metadata"]["key"] == esda.CLEANED_UNTRACKED_KEY
     # Before confirmed_place was introduced, we created confirmed_trips without a confirmed_place
     # For those trips, we will generate a confirmed_place just-in-time and add its ID to the trip
     # Once every trip has a confirmed_place, we can remove this code
-    if "confirmed_place" not in ct["data"]:
+    if not isUntrackedTime and "confirmed_place" not in ct["data"]:
         cleaned_place = esda.get_entry(esda.CLEANED_PLACE_KEY, ct["data"]["end_place"])
-        confirmed_place_entry = create_confirmed_place_entry(ts, cleaned_place)
+        confirmed_place_entry = eaum.create_confirmed_place_entry(ts, cleaned_place)
         cpeid = ts.insert(confirmed_place_entry)
         ct["data"]["confirmed_place"] = cpeid
         logging.debug("Setting the confirmed_place key to the newly created id %s" % cpeid)
         import emission.storage.timeseries.builtin_timeseries as estbt
         estbt.BuiltinTimeSeries.update(ct)
 
-    logging.info("End place type for trip is %s" % type(ct.data.end_place))
+    logging.info("End place type for trip is %s" % type(ct['data']['end_place']))
     composite_trip_dict = copy.copy(ct)
     del composite_trip_dict["_id"]
     composite_trip_dict["metadata"]["key"] = "analysis/composite_trip"
 
-    composite_trip_dict["data"]["confirmed_place"] = get_confirmed_place_for_confirmed_trip(ct)
-    composite_trip_dict["data"]["locations"] = get_locations_for_confirmed_trip(ct)
-    # later we will want to put section & modes in composite_trip as well
+    if not isUntrackedTime:
+        composite_trip_dict["data"]["confirmed_place"] = eaum.get_confirmed_place_for_confirmed_trip(ct)
```

Comment:
Yes, but that place will have a duration of 0, and be ineligible for additions or userinputs, so I did not think we wanted to display it.

Do you think it needs to be shown?

Reply:
I don't think it needs to be shown, but I was wondering if it would break other downstream assumptions about whether or not trips have places. If it will not, I am fine with adding a comment here, and adding test cases for this behavior so that we can be notified if/when it changes.

Reply:
Added comment above.
The Jack Mar 12 test will check the overall output of composite trips, so if something does change, that test can tell us.